code/game/atoms_movable.dm

```diff
@@ -67,15 +70,15 @@
 			return FALSE
 	return ..()
 
-/atom/movable/proc/start_pulling(atom/movable/AM,gs)
+/atom/movable/proc/start_pulling(atom/movable/AM, gs, force = PULL_FORCE_DEFAULT)

```

Comment:
it seems weird to me to pass these through as variables when they're properties that already live on the source and target atoms?

Reply:
also please use a descriptive variable name, instead of `gs`

Reply:
The reason I put them on the procs too is so things can make something act as if it had more force than it would, without doing something like
var/old = move_force
move_force = temporary value
do_thing()
move_force = old

Reply:
I suppose